src/DebugEngineHost/DebugEngineHost.csproj

```diff
@@ -88,6 +88,9 @@
       <IncludeAssets>compile</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+
+    <PackageReference Include="Microsoft.IO.Redist" Version="$(Microsoft_IO_Redist_Version)" />
+    <PackageReference Include="System.Text.Json" Version="$(System_Text_Json_Version)" />
```

Comment:
>/> [](http://example.com/codeflow?start=87&length=2)

`<ExcludeAssets>all</ExcludeAssets>` to make sure we don't actually compile against this?